docs/tutorial/fdfitting.rst

```diff
@@ -0,0 +1,314 @@
+for that particular experimental condition.
+
+It is also possible to obtain simulations from the model directly. We can do this by calling the 
+model with values for the independent variable (here denoted as distance) and the parameters 
+required to simulate the model. Again, we obtain these parameters by grabbing them from our fit
+object using the data handles::
+
+    distance = np.arange(2.0, 5.0, .01)
+    simulation_result = dna_model(distance, odijk_fit.parameters[data1])
+
+Global fits versus single fits
+------------------------------
+
+The `FitObject` manages a fit. To illustrate its use, and how a global fit differs from a
+simple fit, consider the following two examples::
+
+    odijk_inv = pylake.inverted_odijk("DNA")
+    for i, (distance, force) in enumerate(zip(distances, forces)):
+        odijk_inv.load_data(f=force, d=distance, name="RecA")
+    odijk_fit = pylake.FitObject(odijk_inv)
+    odijk_fit.fit()
+    print(odijk_fit.parameters["DNA_Lc"])
+
+and::
+
+    for i, (distance, force) in enumerate(zip(distances, forces)):
+        odijk_inv = pylake.inverted_odijk("DNA")
+        odijk_inv.load_data(f=force, d=distance, name="RecA")
+        odijk_fit = pylake.FitObject(odijk_inv)
+        odijk_fit.fit()
+        print(odijk_fit.parameters["DNA_Lc"])
+
+The difference between these two is that the former sets up a single model, that has to fit
+all the data whereas the latter fits all the datasets independently. The former has one single
+parameter set, whereas the latter has a parameter set per data set. Also note how in the second
+example a new `Model` and `FitObject` is created at every cycle of the for loop.
+
+Statistically, it is typically more optimal to fit data using global fitting, as more
+information goes into estimates of parameters shared between different conditions. It's
+usually a good idea to think about which parameters you expect to be different between
+different experiments and only allow these parameters to be different. For example, if the
+only expected difference between different experiments is the contour length, then this
+can be achieved using::
+
+    odijk_inv = pylake.inverted_odijk("DNA")
+    for i, (distance, force) in enumerate(zip(distances, forces)):
+        odijk_inv.load_data(f=force, d=distance, name="RecA", DNA_Lc=f"DNA_Lc_{i}")
+    odijk_fit = pylake.FitObject(odijk_inv)
+    odijk_fit.fit()
+    print(odijk_fit.parameters)
+
+Note that this piece of code will lead to parameters `DNA_Lc_0`, `DNA_Lc_1` etc.
+
+Incremental fitting
+-------------------
+
+Fits can also be done incrementally::
+
+    >>> odijk_inv = pylake.inverted_odijk("DNA")
+    >>> odijk_fit = pylake.FitObject(odijk_inv)
+    >>> print(odijk_fit.parameters)
+    No parameters
+
+We can see that there are no parameters to be fitted. The reason for this is that
+we did not add any data to the model yet. Let's add some and fit this data::
+
+    >>> data1 = odijk_inv.load_data(f=f1, d=d1, name="Control")
```

Comment:
Nearly all of the examples give data a name. Why not make `name` a required parameter?

Reply:
Yeah. Makes sense.